data/distro.yml

```diff
@@ -7,9 +7,9 @@
         <terminal-command>sudo apt install flatpak</terminal-command>
         <p>With older Ubuntu versions, the official Flatpak PPA is the recommended way to install Flatpak. To install it, run the following in a terminal:</p>
         <terminal-command>
-          sudo add-apt-repository ppa:flatpak/stable\n
```

Comment:
Why the (unrelated) formatting change?

Reply:
I'm sorry, my editor did it automatically, I'm going to send a commit with corrections now